content/doc/book/system-administration/admin-password-reset.adoc

```diff
@@ -0,0 +1,32 @@
+---
+layout: section
+---
+ifdef::backend-html5[]
+:notitle:
+:description:
+:author:
+:email: jenkinsci-users@googlegroups.com
+:sectanchors:
+:toc: left
+endif::[]
+
+==Admin-Password-Reset Jenkins
+As a system administrator, you may need to reset the Jenkins admin password if you have lost or forgotten it. To cover this possibility we have steps to follow to reset the admin password.
+
+==Steps to reset Jenkins Admin Password
```

Comment:
We use sentence case in headings.  Don't need to use Jenkins in the heading, since it is already assumed by the reader.  Separate heading from other text with a blank line is usually a good practice.
```suggestion
== Reset administrator password

```